jena-fuseki2/jena-fuseki-ui/tests/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  env: {
+    mocha: true,
+  },
+}
```

Comment:
Mocha is the test framework. Chai is the assertions library... a pair similar to JUnit Hamcrest I think.